fast/stages/02-networking-vpn/variables.tf

```diff
@@ -50,6 +50,13 @@ variable "data_dir" {
   default     = "data"
 }
 
+variable "data_platform_sa" {
```

Comment:
This will need to use a generic variable, shared by any service account created in stage 1.